src/util/rotary.h

```diff
@@ -2,39 +2,25 @@
 
 #include <vector>
 
+#include "util/assert.h"
+
+// Simple moving average
 class Rotary {
-  public:
-    Rotary();
+    using Buffer = std::vector<double>;
+    using index_type = Buffer::size_type;
 
-    // Start calibration measurement
-    void calibrateStart();
-    // End calibration measurement
-    double calibrateEnd();
-    // Set calibration
-    void setCalibration(double c);
-    // Get calibration
-    double getCalibration();
+  public:
+    Rotary(qsizetype filterLength)
+            : m_filterHistory(filterLength, 0.0) {
```

Comment:
note, that this does restore the previous behavior. I have not yet experimented with more dynamic filtering.